agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2777,25 +2777,6 @@ bool slave_thread::handle_cmdu_monitor_message(Socket *sd,
         send_cmdu_to_controller(cmdu_tx);
         break;
     }
-    case beerocks_message::ACTION_MONITOR_CLIENT_STATISTICS_11K_RESPONSE: {
```

Comment:
This commit should be placed *before* those that remove ACTION_MONITOR_CLIENT_STATISTICS_11K_RESPONSE from TLVF

Reply:
fixed

Reply:
solved